### PR TITLE
[WIP] Integrate Spark docs into the website

### PR DESCRIPTION
**Pull request by tdas**
Cosmetic things that need to be fixed to integrate Spark docs 
- language tabs (find all `.. code-language-tabs::`
- highlighted important info (`.. note::`, `.. important::`, etc.)
- side menu.. needs to be two-level menu.

Other things that need to be fixed
- all links (e.g., rewrite existing markdown format of `[_](link)`)

### CODE REVIEW

**Review by baumanab — COMMENTED** [review]
markdown docs look reasonable, didn't review the menu.js since I'm not qualified.  I suggest we use a workflow where we generate/render the content, review the rendered version(s) and make iterative changes.  We will need a mechanism to keep this process from being user facing.  I'm discussing this with Jim/Denny.

**Comment by jodwyer** [comment]
@tdas with a new change merged last night, you can do this a little better where you can nest items recursively. [Here is an example of what it looks like](https://deploy-preview-37--delta-io-beta.netlify.app/learn) and the code it in this - PR https://github.com/delta-io/website/pull/37

The LinkList code was merged yesterday but and the way you use it is by adding an array of items which you can see in my PR.

Like I said, not exactly what you are looking for but going in the right direction.

File: config/menus.js [hunk 1]
```diff
@@ -101,10 +101,37 @@ const footer = [
...
+  {
+    label: "> Utility operations",
+    url: "/docs/spark/utilities",
+  },
+  {
+    label: "> Constraints",
+    url: "/docs/spark/constraints",
+  },
+];
+
```
Comment (jakebellacera) [inline config/menus.js:129]: Two things:

1. You can nest menu items.
2. Menu items always need a link - you can link to `/docs/spark` for the topmost link.

```suggestion
const docs = [
  {
    label: "Apache Spark",
    url: "/docs/spark",
    items: [
      {
        label: "Batch operations",
        url: "/docs/spark/batch",
      },
      {
        label: "Streaming operations",
        url: "/docs/spark/streaming",
      },
      {
        label: "Update operations",
        url: "/docs/spark/updates",
      },
      {
        label: "Utility operations",
        url: "/docs/spark/utilities",
      },
      {
        label: "Constraints",
        url: "/docs/spark/constraints",
      },
    ],
  },
];

```

**Review by jakebellacera — APPROVED** [review]
I have some feedback on the menu now that we can nest items.

**Pull request closed without merging**